testing/run_all_unittests.cc

```diff
@@ -2,9 +2,22 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#include "flutter/fml/build_config.h"
+#include "flutter/fml/logging.h"
 #include "gtest/gtest.h"
 
+#ifdef OS_IOS
+#include <asl.h>
+#endif  // OS_IOS
+
 int main(int argc, char** argv) {
+#ifdef OS_IOS
+  asl_log_descriptor(NULL, NULL, ASL_LEVEL_NOTICE, STDOUT_FILENO,
```

Comment:
In this harness, both can be ASL_LEVEL_ERR. I don't think notices are shown by default.

Reply:
In a local test they were, but I'll just make them both err.